use-cases/conference-scheduling/src/main/java/org/acme/conferencescheduling/rest/DemoDataGenerator.java

```diff
@@ -38,13 +38,12 @@ public class DemoDataGenerator {
     // Audience tags
     private static final List<String> AUDIENCE_TAGS = List.of("Programmers", "Analysts", "Managers");
     // Content tags
-    private static final List<String> CONTENT_TAGS =
-            List.of("Timefold", "Constraints", "Metaheuristics", "Kubernetes");
-    private final Random random = new Random(0);
+    private static final List<String> CONTENT_TAGS = List.of("Timefold", "Constraints", "Metaheuristics", "Kubernetes");
+    private static final Random random = new Random(0);
 
-    private static final Set<TalkType> TALK_TYPES = Set.of(
+    private static final Set<TalkType> TALK_TYPES = new LinkedHashSet<>(Set.of(
```

Comment:
Not very nice, is it? I recommend adding a new static method `toSet(T... items)`, which would do something like this under the hood:

    var set = new LinkedHashSet<T>(items.length);
    set.addAll(items);
    return Collections.unmodifiableSet(set);

That way, you get reproducibility without sacrificing the readability of your code.